eng/pipelines/templates/steps/analyze_dependency.yml

```diff
@@ -9,7 +9,7 @@ steps:
      versionSpec: '$(PythonVersion)'
 
   - pwsh: |
-     python -m pip install -r eng/ci_tools.txt $(if($IsWindows) {"--user" })
+      python -m pip install -r eng/ci_tools.txt
```

Comment:
100%. I added a todo bullet to the PR body to remember this.

I still need to update all usage in all the builds. There just isn't a build failure yet because of the merge conflict on `shared_requirements.txt`.